src/DeviceController.cpp

```diff
@@ -92,6 +93,19 @@ namespace earlyapp
                 GstVideoDevice::getInstance() : pVid;
             m_pCam = (pVid == nullptr) ?
```

Comment:
This line is redundant due to the else statement below.

Reply:
yes, will remove this line because using in else statement